### PR TITLE
Add Github docs files

### PR DESCRIPTION
**Pull request by micschwarz**
- License
- Pull Reqest Tpl
- Contributing

### CODE REVIEW

File: docs/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (jens1o) [inline docs/LICENSE.md]: this file should be in the root folder instead

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+###Changlog
+- foo
+- bar
+- baz
```
Comment (jens1o) [inline docs/PULL_REQUEST_TEMPLATE.md]: incomplete

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Code
+* Be conform with tslint settings
+* Set linebreaks to be CRLF
+
+# Commits
+* Be conform with [this Guide](https://chris.beams.io/posts/git-commit/)
```
Comment (jens1o) [inline docs/CONTRIBUTING.md]: lol, that's not that important actually.

Comment (micschwarz) [reply docs/CONTRIBUTING.md]: It is.

File: docs/LICENSE.md (hunk 1 above)
Comment (micschwarz) [reply docs/LICENSE.md]: ok

File: docs/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (micschwarz) [reply docs/PULL_REQUEST_TEMPLATE.md]: why?

Comment (jens1o) [reply docs/PULL_REQUEST_TEMPLATE.md]: Ah... Now I get it. It should be `Changelog` then.

File: docs/CONTRIBUTING.md (hunk 3 above)
Comment (jens1o) [reply docs/CONTRIBUTING.md]: 😝

File: docs/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (jens1o) [reply docs/PULL_REQUEST_TEMPLATE.md]: GitHub flavored-markdown needs a space after the `#`

**Pull request merged**